File: ssaExample/ssa_Function.java
import java.util.Map;

public class ssa_Function {

	public String name;
	public types_Func object;
	public ssa_selection method;
	public types_Signature Signature;
	public Integer pos;
	public String Synthetic;
	public Object syntax;
	public types_Info info;
	public String goversion;
	public ssa_Function parent;
	public ssa_Package Pkg;
	public ssa_Program Prog;
	public ssa_Parameter[] Params;
	public ssa_FreeVar[] FreeVars;
	public ssa_Alloc[] Locals;
	public ssa_BasicBlock[] Blocks;
	public ssa_BasicBlock Recover;
	public ssa_Function[] AnonFuncs;
	public Object[] referrers;
	public Integer anonIdx;
	public types_TypeParamList typeparams;
	public Object[] typeargs;
	public ssa_Function topLevelOrigin;
	public ssa_generic generic;
	public ssa_BasicBlock currentBlock;
	public Map<types_Var, Object> vars;
	public ssa_Alloc[] namedResults;
	public ssa_targets targets;
	public Map<types_Label, ssa_lblock> lblocks;
	public ssa_subster subst;
}

File: ssaExample/ssa_Function.kt
class ssa_Function {

	var name: String? = null
	var Object: types_Func? = null
	var method: ssa_selection? = null
	var Signature: types_Signature? = null
	var pos: Int? = null
	var Synthetic: String? = null
	var syntax: Any? = null
	var info: types_Info? = null
	var goversion: String? = null
	var parent: ssa_Function? = null
	var Pkg: ssa_Package? = null
	var Prog: ssa_Program? = null
	var Params: Array<ssa_Parameter>? = null
	var FreeVars: Array<ssa_FreeVar>? = null
	var Locals: Array<ssa_Alloc>? = null
	var Blocks: Array<ssa_BasicBlock>? = null
	var Recover: ssa_BasicBlock? = null
	var AnonFuncs: Array<ssa_Function>? = null
	var referrers: Array<Any>? = null
	var anonIdx: Int? = null
	var typeparams: types_TypeParamList? = null
	var typeargs: Array<Any>? = null
	var topLevelOrigin: ssa_Function? = null
	var generic: ssa_generic? = null
	var currentBlock: ssa_BasicBlock? = null
	var vars: Map<types_Var, Any>? = null
	var namedResults: Array<ssa_Alloc>? = null
	var targets: ssa_targets? = null
	var lblocks: Map<types_Label, ssa_lblock>? = null
	var subst: ssa_subster? = null
}
